packages/vscode-extension/src/resources/Strings.json

```diff
@@ -74,7 +74,9 @@
       "previewAdaptiveCard": "Preview and Debug Adaptive Cards",
       "previewACDesciption": "Preview and author Adaptive Cards directly in Visual Studio Code",
       "grantPermissionSucceeded": "Added account: '%s' to the environment '%s' as a collaborator.",
-      "grantPermissionWarning": "You need to handle Azure resource permissions manually on Azure Portal.",
+      "grantPermissionWarningSpfx": "If added user is not a SharePoint App Catalog site admin, you need to handle that via SharePoint admin center. ",
+      "grantPermissionWarning": "If added user cannot access Azure resources, you need to handle that via Azure portal. ",
```

Comment:
-> "..., you need to manually setup access policy via Azure Portal"?

Reply:
@MuyangAmigo , what do you think about the message with Bowen's suggestion?

`If added user cannot access Azure resources, you need to manually setup access policy via Azure Portal`